lua/entities/gmod_wire_expression2/core/custom/stcontrol.lua

```diff
@@ -573,79 +600,79 @@ end
 
 __e2setcost(3)
 e2function array stcontrol:getGain()
-	if not this then return {0,0,0} end
+	if(not this) then return {0,0,0} end

```

Comment:
I don't see the purpose in adding these parentheses.

Reply:
It is prettier this way. And 1:1 with the main repository.

Reply:
It's prettier for C users, but not lua, but this is your own library so I don't mind it